models/upload_sessions.go

```diff
@@ -2,14 +2,15 @@ package models
 
```

Comment:
There's some commented out lines in this file that we could remove but that's no biggie.